### PR TITLE
adding apm sub config module

### PR DESCRIPTION
**Pull request by peterBrxwn**

### CODE REVIEW

File: Countly.d.ts [hunk 1]
```diff
@@ -1213,3 +1218,41 @@ declare module "countly-sdk-react-native-bridge/CountlyConfig" {
 
   export default CountlyConfig;
 }
+
+declare module "countly-sdk-react-native-bridge/lib/configuration_interfaces/countly_config_apm" {
```
Comment (turtledreams) [inline Countly.d.ts:1222]: idt you need to declare it. Cant u add a nested interface on countly config?

**Review by turtledreams — CHANGES_REQUESTED** [review]

File: Countly.d.ts (hunk 1 above)
Comment (peterBrxwn) [reply Countly.d.ts]: done.

**Comment by peterBrxwn** [comment]
[[RN] Rework init time configuration options.](https://trello.com/c/AlaiN13m/1464-rn-rework-init-time-configuration-options)

**Comment by turtledreams** [comment]
Okay I did some basic testing and all seems to be good. Except line 133 at CountlyReactNative.java. An old enableManualAppLoadedTrigger was there. I am removing it now.

**Review by turtledreams — CHANGES_REQUESTED** [review]
I dont like methods and properties to have the same name. Shall we change it?
image

**Comment by turtledreams** [comment]
or better lets remove them from d.ts file as it has no purpose for the end user
image

**Review by turtledreams — APPROVED** [review]

**Comment by peterBrxwn** [comment]
[[RN] remove any changes to the package lock file from apm_config PR](https://trello.com/c/ho9pgUdB/1512-rn-remove-any-changes-to-the-package-lock-file-from-apmconfig-pr)

**Pull request merged**